packages/reactive-element/src/test/std-decorators/property_test.ts

```diff
@@ -161,22 +161,36 @@ suite('@property', () => {
         return this._foo as number;
       }
 
-      override updated() {
-        this.updatedContent = this.foo;
+      override updated(changedProperties: PropertyValues<this>) {
+        this.updatedProperties = changedProperties;
       }
     }
     customElements.define(generateElementName(), E);
     const el = new E();
     container.appendChild(el);
     await el.updateComplete;
+
+    // Check initial values
     assert.equal(el._foo, undefined);
-    assert.equal(el.updatedContent, undefined);
+    assert.equal(el.updatedProperties!.get('foo'), undefined);
```

Comment:
Here and in the main decorators tests: should we assert on `el.updatedProperties!.has('foo')` here?

Reply:
Since there's no initializer it'll be false, but I can assert on that instead